x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/agent_config/create_package_config_page/step_select_config.tsx

```diff
@@ -23,16 +25,35 @@ import {
   useGetAgentConfigs,
   sendGetOneAgentConfig,
   useCapabilities,
+  useFleetStatus,
 } from '../../../hooks';
 import { CreateAgentConfigFlyout } from '../list_page/components';
 
+const AgentConfigWrapper = styled(EuiFormRow)`
+  .euiFormRow__label {
+    width: 100%;
+  }
+`;
+
+const AgentConfigNameColumn = styled(EuiFlexItem)`
+  max-width: ${(props) => `${((props.grow as number) / 9) * 100}%`};
```

Comment:
Why is this additional calculation needed? Would it make sense to add a comment about it?

Reply:
1) the `max-width` property is added to prevent long config names/descriptions from overflowing the flex items
2) we build it from the `grow` property on the flex items because that property changes based on if Fleet is enabled/setup or not (when Fleet is not available, the "agents enrolled" column is not displayed)

Reply:
pushed up a change with comments added